src/prototype_creep_resources.js

```diff
@@ -364,6 +364,10 @@ const filterTransferrables = function(creep, object) {
     return false;
   }
 
+  if (object.structureType === STRUCTURE_STORAGE && _.sum(object.store) + _.sum(creep.carry) > object.storeCapacity) {
```

Comment:
you should place it with l 363 for codeclimate: 
```
object.structureType === STRUCTURE_STORAGE ? 
_.sum(object.store) + _.sum(creep.carry) > object.storeCapacity :
object.energy === object.energyCapacity

```

Reply:
thx